test/global.test.ts

```diff
@@ -11,7 +11,6 @@ export const longRunningTestsEnabled: boolean = !/^(false|0)?$/i.test(process.en
 
 // Runs before all tests
 suiteSetup(async function (this: Mocha.Context): Promise<void> {
-    this.skip();
```

Comment:
I think that we may still want to skip the long running tests. I'm pretty sure that we haven't converted the Azure account credentials to be compatible.

Reply:
I explicitly set the long running tests flag to false.